docs/docs/noir_js/reference/01_noirjs.md

```diff
@@ -81,6 +84,7 @@ async execute(inputs)
 
 ```js
 const { witness, returnValue } = await noir.execute(inputs)
+const { witness, returnValue } = await noir.execute(inputs, (event) => console.log(event))
```

Comment:
```suggestion
const { witness, returnValue } = await noir.execute(inputs, (name, args) => console.log(`Received foreign call ${name} with arguments ${args}`))
```

We should use the correct function signature here. I don't think that example would compile.